packages/storybook/src/nimble/mapping/empty/mapping-empty.stories.ts

```diff
@@ -0,0 +1,35 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { mappingKeyDescription } from '../base/story-helpers';
+import { createUserSelectedThemeStory } from '../../../utilities/storybook';
+import { hiddenWrapper } from '../../../utilities/hidden';
+
+const metadata: Meta = {
+    title: 'Internal/Mappings',
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    `The \`nimble-mapping-empty\` element defines a mapping from a data value to text. It is meant to be used as content of the \`nimble-table-column-mapping\` element
+                    when the table cell should not include the mapped value and the column is groupable.`
```

Comment:
This mapping type is less obvious than others, so I think we should be more explicit about why a client would use this.
```suggestion
                    `The \`nimble-mapping-empty\` element can be provided as content of \`nimble-table-column-mapping\` to define data values that should render as empty table cells, but should be distinctly groupable. For each such data value, provide a \`nimble-mapping-empty\` element that maps the value to a distinct text value to group by.`
```